### PR TITLE
Cleanup Android Howto doc

### PR DESCRIPTION
**Pull request by MarkY-LunarG**
Apply Erika's comments and wordsmithing to the doc.

### CODE REVIEW

File: HOWTO_android.md [hunk 1]
```diff
@@ -20,23 +20,23 @@ Copyright &copy; 2023 LunarG, Inc.
...
 
 ## Building Vulkan Samples with the Capture Layer
 
-Let's look at adding the GFXReconstruct layer as a requirement of the 
+Let's look at adding the GFXReconstruct layer as a requirement of the
 [Khronos Vulkan Samples](https://github.com/KhronosGroup/Vulkan-Samples)
 project.
-These steps assume that you have all the dependencies necessary to build that
-project, so refer to that repo to ensure those are met before continuing.
+Completing the following steps will ensure that you have all the dependencies
```
Comment (bradgrantham-lunarg) [inline HOWTO_android.md:37]: I think your original thought was correct - the following steps all assume you have met all the dependencies to build Vulkan-Samples.

**Review by bradgrantham-lunarg — CHANGES_REQUESTED** [review]
LGTM except for the one thought about dependencies

File: HOWTO_android.md (hunk 1 above)
Comment (MarkY-LunarG) [reply HOWTO_android.md:37]: I'll revert it

Comment (MarkY-LunarG) [reply HOWTO_android.md]: I cleaned it up.  Not a direct revert, but near enough.

**Pull request merged**